sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -168,19 +168,22 @@ private void CloseLink(ReceivingAmqpLink link)
         /// Receives a batch of <see cref="ServiceBusReceivedMessage" /> from the entity using <see cref="ReceiveMode"/> mode.
         /// </summary>
         ///
-        /// <param name="maximumMessageCount">The maximum number of messages that will be received.</param>
+        /// <param name="maxMessages">The maximum number of messages that will be received.</param>
+        /// <param name="maxWaitTime">An optional <see cref="TimeSpan"/> specifying the maximum time to wait for the first message before returning an empty list if no messages have been received.</param>
         /// <param name="cancellationToken">An optional <see cref="CancellationToken"/> instance to signal the request to cancel the operation.</param>
         ///
         /// <returns>List of messages received. Returns null if no message is found.</returns>
         public override async Task<IList<ServiceBusReceivedMessage>> ReceiveBatchAsync(
-            int maximumMessageCount,
+            int maxMessages,
```

Comment:
maxMessages -> maxMessageCount ?

Reply:
We use maxMessages in the public API.

Reply:
yeah, I meant can we rename the public API to use maxMessageCount?

Reply:
I'm not sure the Count adds much as maxMessages seems pretty clear IMO.

Reply:
I'm fine either way but to me maxMessageCount seems more suitable in this context.

Reply:
I don't have a strong preference on this either, but I assumed this was out of scope for this PR. I only updated this parameter name for the internal method to be consistent with the other uses. I'd say we can look to see if there are guidelines or examples in other libraries where they append Count, but we can do this for a later preview.